src/utilities/theme/utils.ts

```diff
@@ -22,18 +22,18 @@ export function buildCustomProperties(
   newDesignLanguage: boolean,
   tokens?: Record<string, string>,
 ): CustomPropertiesLike {
-  const {colors = {}, colorScheme, config, frameOffset = '0px'} = themeConfig;
+  const {colors = {}, colorScheme, config, frameOffset = 0} = themeConfig;
   const mergedConfig = mergeConfigs(base, config || {});
 
   return newDesignLanguage
     ? customPropertyTransformer({
         ...colorFactory(colors, colorScheme, mergedConfig),
         ...tokens,
-        frameOffset,
+        frameOffset: `${frameOffset}px`,
       })
     : {
         ...buildLegacyColors(themeConfig),
-        ...(frameOffset ? customPropertyTransformer({frameOffset}) : null),
```

Comment:
This is an optional prop but I'm providing a default of 0 so I don't need the ternary. I'm also doing the interpolation to add `px` in TS vs CSS which seems like it would be messy @tmlayton